test/framework/cluster.go

```diff
@@ -921,11 +921,13 @@ func (e *ClusterE2ETest) SetPackageBundleActive() {
 	if err != nil {
 		e.T.Fatalf("Error getting PackageBundle: %v", err)
 	}
+	os.Setenv("KUBECONFIG", kubeconfig)
 	if pbc.Spec.ActiveBundle != pb[0].ObjectMeta.Name {
 		e.RunEKSA([]string{
 			"upgrade", "packages",
 			"--bundle-version", pb[0].ObjectMeta.Name, "-v=9",
 			"--cluster=" + e.ClusterName,
+			"--kubeconfig" + e.kubeconfigFilePath(),
```

Comment:
These values are the same. Do we need to set the env and pass in the variable?

Reply:
probably not, but I'm not sure which one should take precedent here. Maybe I'll just remove the flag, and keep the ENV var.

Reply:
Fixed in [a9154b7](https://github.com/aws/eks-anywhere/pull/4254/commits/a9154b79d80196a52f479fa590680420903ea3e9)